### PR TITLE
7501 - Tabbed Idle Logout Bug

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

**Review by rachaelparris — COMMENTED** [review]
Was the file `web-client/src/presenter/actions/batchDownloadTrialSessionAction.test.js` removed accidentally?

File: web-client/src/presenter/actions/broadcastIdleStatusActiveAction.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/**
+ * broadcasts an active state to all app instances
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.get the cerebral get function to retrieve state values
+ * @returns {number} total computed value from penalty values
```
Comment (rachaelparris) [inline web-client/src/presenter/actions/broadcastIdleStatusActiveAction.js]: Nit: jsdocs

File: web-client/src/presenter/actions/broadcastStayLoggedInAction.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/**
+ * broadcasts a stay logged in message to all app instances
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.applicationContext application context used to get the broadcast gateway
+ * @returns {number} total computed value from penalty values
```
Comment (rachaelparris) [inline web-client/src/presenter/actions/broadcastStayLoggedInAction.js]: Nit: jsdocs

**Review by rachaelparris — APPROVED** [review]
Just a few nits. Great work!

**Comment by kkoskelin** [comment]
> Was the file `web-client/src/presenter/actions/batchDownloadTrialSessionAction.test.js` removed accidentally?

Definitely a mistake when doing some other cleanup. Will fix!

**Pull request merged**